source/browseMode.py

```diff
@@ -501,7 +501,7 @@ def maybeSyncFocus(self, postFocusFunc=None):
 			synchronousCall = True
 
 		if not synchronousCall:
-			uid = obj.uniqueID
+			uid = obj._get_uniqueID()

```

Comment:
obj.uniqueID should still work here. the ```_get_*``` methods are magic properties :)

Reply:
Got it. So many fancy features in Python!